src/CartList.js

```diff
@@ -3,28 +3,54 @@ import { ScrollView, Text, View } from 'react-native';
 import { GlobalConsumer } from 'store/GlobalProvider';
 import CartArticle from 'CartArticle';
 import Article from 'Article';
-import { Button } from 'react-native-elements';
+import { Button, Card } from 'react-native-elements';
 import call from 'react-native-phone-call';
 
-const CartList = () => {
-  // Note Louis : pour faire cette fonction tu peux utiliser reduce des Array
-  getTotalPrice = cart => 10;
+const CartList = (props) => {
+  // Utiliser reduce des Array
+  getTotalPrice = (cart) => {
+    totalPrice = 0.0;
+    cart.forEach(function (articleInCart) {
```

Comment:
arrow function ? ^^
Tu aurais pu aussi utiliser reduce mais pas de soucis
```suggestion
    cart.forEach( (articleInCart) => {
```

Reply:
J'avais regardé le reduce mais j'avais pas réussi à faire un truc qui fonctionnait..

Reply:
Oui c'est p'tet un peu compliqué avec un array d'objets ^^
Pas de soucis, toute façon j'ai mergé dans develop :-)